
ut_alpha: 1.0

# state transition model 
/rmsgf/linear_sigma: 0.002
/rmsgf/angular_sigma: 0.01
/rmsgf/velocity_factor: 0.8

# observation model 
obsrv_fg_noise_std: 0.001

# uniform tail arguments
obsrv_body_tail_weight: 0.1
uniform_tail_min: -5000.0
uniform_tail_max: 5000.0

obsrv_bg_depth: -3.0
obsrv_bg_noise_std: 100.1

# object colors
gt_color_r: 97
gt_color_g: 117
gt_color_b: 123
belief_color_r: 250
belief_color_g: 232
belief_color_b: 184

